Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs

```diff
@@ -35,9 +35,15 @@ public TestFramework(string vsixScriptFolder)
         {
             var testInfo = string.Empty;
             var discoverResultFile = Path.GetTempFileName();
+
+            // Create a temp file with the list of test files.
+            var testFilesString = string.Join(";", testFiles);
+            var testFilesListFile = Path.GetTempFileName();
+            File.AppendAllText(testFilesListFile, testFilesString, System.Text.Encoding.UTF8);
+
             try
             {
-                var stdout = EvaluateJavaScript(nodeExe, string.Join(";", testFiles), discoverResultFile, logger, projectRoot);
+                var stdout = EvaluateJavaScript(nodeExe, testFilesListFile, discoverResultFile, logger, projectRoot);
```

Comment:
Is this the only call to `EvaluateJavaScript`?

Reply:
Yes it's the only call, I searched for all references in the project.